### PR TITLE
Add processorTest and testCommon modules.

### PR DESCRIPTION
**Pull request by ivanisidrowu**
## Added
- Add `processorTest` module to test the annotation processor and parser generators.
- Add `testCommon` module to include those RSS XML files and extensions for testing. (`ktrssreader` and `processorTest` can use it.)

## Modified
- Move common RSS XML files and extensions to new module, `testCommon`.

## TODO
- More parser generator tests.

### CODE REVIEW

File: processorTest/src/test/java/tw/ktrssreader/processor/test/ExampleUnitTest.kt [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
```
Comment (dev-weiqi) [inline processorTest/src/test/java/tw/ktrssreader/processor/test/ExampleUnitTest.kt:28]: 這個 file 應該可以不用上

**Review by dev-weiqi — COMMENTED** [review]
Done!

File: processorTest/src/test/java/tw/ktrssreader/processor/test/ExampleUnitTest.kt (hunk 1 above)
Comment (ivanisidrowu) [reply processorTest/src/test/java/tw/ktrssreader/processor/test/ExampleUnitTest.kt]: done

**Review by dev-weiqi — APPROVED** [review]
🤟🏿

File: processorTest/build.gradle [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: "libs", include: ["*.jar"])
```
Comment (HsienDaDa) [inline processorTest/build.gradle:45]: 印象是沒有放額外的lib在libs folder, 其實這行可以mark掉節省點拉code的時間

Comment (ivanisidrowu) [reply processorTest/build.gradle]: done

**Review by HsienDaDa — APPROVED** [review]

**Review by AmyPlace — APPROVED** [review]
LGTM

**Pull request merged**